# In Python 3.10, the abstract base class (ABC) MutableMapping
# was moved from the 'collections' module to 'collections.abc'.
# Reference: https://docs.python.org/3.9/library/collections.html
#
--- a/share/support/core/gs_utils/__init__.py
+++ b/share/support/core/gs_utils/__init__.py
@@ -2,7 +2,7 @@
 # exported by GPS to make it easier to write plugins

 from GPS import pwd, cd, Action, EditorBuffer, MDI
-from collections import MutableMapping as DictMixin
+from collections.abc import MutableMapping as DictMixin
 import types
 import GPS
 import GPS.Browsers
